vllm/model_executor/models/deepseek_v2.py

```diff
@@ -500,8 +498,6 @@ def forward(
         self,
         positions: torch.Tensor,
         hidden_states: torch.Tensor,
-        kv_cache: torch.Tensor,
-        attn_metadata: AttentionMetadata,
```

Comment:
for updating deepseek_v1 by passing kv_cache and attn_metadata using forward_context, suggest to use another PR to do the update. I assume this is not related to MTP?

Reply:
Let me check if I can revert this interface change.